docs/alma/alma.rst

```diff
@@ -178,6 +178,25 @@ or if you wanted all projects by a given PI:
 
     >>> Alma.query(payload=dict(pi_name='Ginsburg, Adam'))
 
+Querying For Proprietary Data
+-----------------------------
+
+To query for proprietary data (non-public data), you need to first authenticate using the ``login`` method,
+and then set ``public=False`` in your query:
+
+.. doctest-skip::
+
+    >>> from astroquery.alma import Alma
+    >>> alma = Alma()
+    >>> # First login to access proprietary data
+    >>> alma.login("username")  # Will prompt for password
+    >>> # Now query including proprietary data
+    >>> proprietary_data = alma.query(payload=dict(project_code='2017.1.01355.L', public=False))
```

Comment:
Isn't it `public=None` to include both public and proprietary?

Reply:
yes, this is definitely a mistake